lib/l10n/app_en.arb

```diff
@@ -507,6 +507,7 @@
   "toStudy": "Study",
   "importGame": "Import game",
   "importGameExplanation": "Paste a game PGN to get a browsable replay,\ncomputer analysis, game chat and shareable URL.",
+  "importGameCaveat": "Variations will be erased. To keep them, import the PGN via a study.",
```

Comment:
Have you modified this file by hand? You should not do that, it is generated automatically.